src/test/scala/uk/gov/nationalarchives/tdr/validation/MetadataValidationJsonSchemaSpec.scala

```diff
@@ -106,6 +106,64 @@ class MetadataValidationJsonSchemaSpec extends TestKit(ActorSystem("MetadataVali
     }
   }
 
+  "MetadataValidationJsonSchema CLOSURE_SCHEMA" should {
+
+    "not return any errors when closure_type is Open" in {
+      val data: Set[ObjectMetadata] = Set(ObjectMetadata("file1", Set(Metadata("closure_type", "Open"))))
+      val validationErrors = MetadataValidationJsonSchema.validate(CLOSURE_SCHEMA, data)
+      validationErrors("file1") shouldBe List.empty
+    }
+
+    "not return any errors when closure_type is Closed" in {
+      val data: Set[ObjectMetadata] = closureDataBuilder("1990-01-12", "33", "12", "1990-11-12", "false", "false")
+      val validationErrors = MetadataValidationJsonSchema.validate(CLOSURE_SCHEMA, data)
+      validationErrors("file1") shouldBe List.empty
+    }
+
+    "return errors with all the required fields when closure_type is Closed" in {
+      val closure = Metadata("closure_type", "Closed")
+      val data: Set[ObjectMetadata] = Set(ObjectMetadata("file1", Set(closure)))
+      val validationErrors = MetadataValidationJsonSchema.validate(CLOSURE_SCHEMA, data)
+      validationErrors("file1").size shouldBe 6
+      validationErrors("file1") should contain theSameElementsAs List(
+        Error("closure_start_date", "required"),
+        Error("foi_exemption_code", "required"),
+        Error("closure_period", "required"),
+        Error("foi_exemption_asserted", "required"),
+        Error("description_closed", "required"),
+        Error("title_closed", "required")
+      )
+    }
+
+    "return errors if required fields has invalid values when closure_type is Closed" in {
+      val data: Set[ObjectMetadata] = closureDataBuilder("1990--12", "55", "-12", "1990--12", "tttt", "tttt")
+      val validationErrors = MetadataValidationJsonSchema.validate(CLOSURE_SCHEMA, data)
+      validationErrors("file1").size shouldBe 6
+      validationErrors("file1") should contain theSameElementsAs List(
+        Error("closure_start_date", "format.date"),
+        Error("foi_exemption_code", "enum"),
+        Error("closure_period", "minimum"),
+        Error("foi_exemption_asserted", "format.date"),
+        Error("description_closed", "enum"),
```

Comment:
description closed and title closed errors should be `type`

Reply:
It returns `eum` only if the value doesn't match with the defined values.

Reply:
Changed `description_closed` and `title_closed` from enum to boolean and it’s returning now the `type` error. Thanks